src/Elewant/Trading/DomainModel.md

```diff
@@ -0,0 +1,84 @@
+Welcome to the Trading domain
+-----------------------------
+
+
+### Offers
+
+Here, we start with an `Offer`. An `Offer` is provided by a `seller`, who is represented here by nothing
+more than a `SellerId`.
```

Comment:
How is this different than a `shepherdId`?

Reply:
It's not really - meaning we will get the same GUID string. But since a "Shepherd" is not really involved in the Trading Bounded Context, and since there will be two parties involved (Seller and Buyer) that are both identified by a `ShepherdId` in Herding, we want to prevent confusion.